README.md

```diff
@@ -18,12 +17,13 @@ Sbt-restli is made up of 4 individual plugins.
 
 ### RestliSchemaPlugin
 
-The rest.li schema plugin compiles pegasus data-schemas (`.pdsc` files) into java data-templates.
+The rest.li schema plugin compiles pegasus data-schemas (`*.pdsc` files) into Java data-templates classes.
 
-Apply the plugin to your project and place data-schemas in the `src/main/pegasus` directory.
+Apply the plugin to your project in `build.sbt` and place data-schemas in the `src/main/pegasus` directory.
 
 ```scala
-// build.sbt
+// TODO - add import for the plugin
```

Comment:
The `autoImport` object inside auto plugins is automatically glob imported by sbt, so no imports are needed from the end user. 

```suggestion
```